code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/MapsActivity.java

```diff
@@ -0,0 +1,210 @@
+package com.example.habittracker.activities.eventlist;
```

Comment:
This whole file also needs java doc comments

Reply:
Fixed